packages/fluent-theme/src/components/suggestedActions/SuggestedAction.tsx

```diff
@@ -1,10 +1,11 @@
 import { hooks } from 'botframework-webchat-component';
 import { type DirectLineCardAction } from 'botframework-webchat-core';
 import cx from 'classnames';
-import React, { MouseEventHandler, memo, useCallback, useRef } from 'react';
+import React, { MouseEventHandler, memo, useCallback } from 'react';
```

Comment:
Not your fault. No need to fix for now. We will have a huge push to clean up imports.

```suggestion
import React, { type MouseEventHandler, memo, useCallback } from 'react';
```

Reply:
Mine PR on code hygiene has a lint rule enforcing this.